### PR TITLE
Funnelweb: init at 3.20

### PR DESCRIPTION
**Pull request by AndersonTorres**

### CODE REVIEW

File: pkgs/development/tools/literate-programming/funnelweb/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  installPhase = ''
+    install -d $out/bin
+    install fw $out/bin/fw
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A simple, reliable literate-programming macro preprocessor";
+    homepage = http://www.ross.net/funnelweb/;
+    license = licenses.gpl2;
```
Comment (cillianderoiste) [inline pkgs/development/tools/literate-programming/funnelweb/default.nix]: Please add yourself as the maintainer.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -5659,6 +5659,8 @@ let
 
   noweb = callPackage ../development/tools/literate-programming/noweb { };
   nuweb = callPackage ../development/tools/literate-programming/nuweb { tex = texlive.combined.scheme-small; };
+  funnelweb = callPackage ../development/tools/literate-programming/funnelweb { };
```
Comment (cillianderoiste) [inline pkgs/top-level/all-packages.nix]: Packages should have a newline between then and be ordered alphabetically.

**Comment by AndersonTorres** [comment]
Yep! I will fix it.

**Comment by cillianderoiste** [comment]
Thanks, I see the order in all-packages still isn't quite right, I'll fix it up (and for eweb) after merging.

**Pull request merged**